mobile/Ihc/screens/PatientHomeScreen.js

```diff
@@ -21,49 +27,97 @@ class PatientHomeScreen extends Component<{}> {
   constructor(props) {
     super(props);
     this.props.navigator.setOnNavigatorEvent(this.onNavigatorEvent.bind(this));
+    this.state = {
```

Comment:
It seems like this really large state is the same as other screens. Can we make a new class that has all of these fields, and then the state just holds an instance of that class?